### DISCUSSION TITLE
Subet seurat based on gene list but: .subscript.2ary(x, i, j, drop = TRUE) : subscript out of bounds

### DISCUSSION THREAD

**Discussion by dduneau** [satijalab/seurat#8930] [Q&A]
Hello,

I have updated my version of Seurat and since then I am unable to subset my seurat object. 
Before I would have wrote:

Seurat_obj_subset = subset(Seurat_obj, features==Genes_selected_expression)

but now this return erro in .subscript.2ary(x, i, j, drop = TRUE) : subscript out of bounds

I know for certain that the genes are present in my seurat object. 
I have tried different ways including the function proposed here : https://github.com/alikhuseynov/add-on_R/blob/develop/R/subset_obj_seurat_v2.R

but I always get the same message.
What I find surprising is that this example is not in the help (https://satijalab.org/seurat/articles/essential_commands.html) while it seems relatively trivial to wish to subset a seurat object to some genes.

Thank you for your help!

Best,
David

**Comment by clarapereira** [comment]
In case anyone else is having this issue, the following worked for me:

`seurat_obj[["RNA"]] <- subset(seurat_obj[["RNA"]], features = select_genes)`
